rst_files/fundamental_types.rst

```diff
@@ -449,7 +449,7 @@ As you can see, the type is ``2×2 Diagonal{Float64,Array{Float64,1}}``, which i
 
 The reasons for this are both efficiency in storage, as well as efficiency in arithmetic and matrix operations
 
-For example, this both behaves (and is, in an important sense) like any other Matrix
+However, this object still behaves like any other ``Matrix``
```

Comment:
```suggestion
This object isn't a ``Matrix`` proper, but it behaves like one because Julia recognizes it as a special example of one (see the `introduction to types lecture <introduction_to_types>`_ for more)
```